content/rooms/fragments/v11-redactions.md

```diff
@@ -0,0 +1,38 @@
+---
+---
+
+{{% added-in this=true %}} The top-level `origin`, `membership`, and `prev_state` properties
+are no longer protected from redaction. The `m.room.create` event no longer keeps the `creator`
+property and now keeps the entire `content` property. The `m.room.redaction` event keeps the
```

Comment:
well, it does preserve `creator`, if there is one, because it preserves all the content.
```suggestion
are no longer protected from redaction. The `m.room.create` event now keeps the entire `content` property. The `m.room.redaction` event keeps the
```